src/main/java/nl/ordina/jobcrawler/model/Skill.java

```diff
@@ -9,12 +9,13 @@
 import java.util.Set;
 import java.util.UUID;
 
+//@Data gives StackOverflowError
```

Comment:
That's because of the bi-directional link between Skill and Vacancy. I don't think there's anything we can do about that at the moment.